docker/Dockerfile

```diff
@@ -44,19 +45,38 @@ RUN apt install -y --no-install-recommends \
     sudo
 RUN update-ca-certificates
 
+COPY ./docker/guest/run_services.sh /root/
+RUN chmod +x /root/run_services.sh
+
+
+FROM runner-base AS runner-with-local-source
+
 # Install CF debian packages.
-COPY --from=cuttlefish-hostpkg-builder /root/android-cuttlefish/cuttlefish-*.deb /root/debian/
+COPY --from=host-package-builder /root/android-cuttlefish/cuttlefish-*.deb /root/debian/
 RUN apt install -y --no-install-recommends -f \
     /root/debian/cuttlefish-base_*.deb \
     /root/debian/cuttlefish-user_*.deb \
     /root/debian/cuttlefish-orchestration_*.deb
 
 RUN echo "num_cvd_accounts=100" >> /etc/default/cuttlefish-host-resources
-
 RUN usermod -aG kvm root
 RUN usermod -aG cvdnetwork root
+ENTRYPOINT ["/root/run_services.sh"]
 
-COPY --from=cuttlefish-hostpkg-builder /root/android-cuttlefish/docker/guest/run_services.sh /root/
-RUN chmod +x /root/run_services.sh
-RUN rm -rf /root/android-cuttlefish
+
+FROM runner-base AS runner-with-apt-install
+
+RUN apt install -y --no-install-recommends \
+    wget
+RUN wget -qO- https://artifacts.codelinaro.org/artifactory/linaro-372-googlelt-gigabyte-ampere-cuttlefish-installer/gigabyte-ampere-cuttlefish-installer/latest/debian/linaro-glt-gig-archive-bookworm.asc \
+    | tee /etc/apt/trusted.gpg.d/linaro-glt-gig-archive-bookworm.asc
+RUN echo "deb https://artifacts.codelinaro.org/linaro-372-googlelt-gigabyte-ampere-cuttlefish-installer/gigabyte-ampere-cuttlefish-installer/latest/debian bookworm main" \
+    | tee /etc/apt/sources.list.d/linaro-glt-gig-archive-bookworm.list
+RUN apt update
+RUN apt install -y --no-install-recommends \
+    cuttlefish-base cuttlefish-user cuttlefish-orchestration
+
+RUN echo "num_cvd_accounts=100" >> /etc/default/cuttlefish-host-resources
```

Comment:
these block is duplicated, how about adding another stage for that?

```
FROM runner-base AS runner-with-local-source
// install debs from host-package-builder
FROM runner-base AS runner-with-apt-install
// install via apt

FROM runner-with-${build_option} AS runner
RUN echo "num_cvd_accounts=100" >> /etc/default/cuttlefish-host-resources
RUN usermod -aG kvm root
RUN usermod -aG cvdnetwork root
ENTRYPOINT ["/root/run_services.sh"]
```

Reply:
Thanks for the suggestion. It's nice! Done :)